app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -95,7 +95,7 @@ $include-html-global-classes: $include-html-classes;
 // $font-weight-normal: normal;
 // $font-weight-bold: bold;
 
-// $white       : #FFFFFF;
+$white       : #FFFFFF;
```

Comment:
Color `#FFFFFF` should be written as `#ffffff`<br>Variable names should be followed immediately by a colon